.github/workflows/main.yaml

```diff
@@ -0,0 +1,307 @@
+name: Build and Deploy Image
+
+on:
+  workflow_dispatch:
```

Comment:
No inputs - So there is no way to manually build/deploy from specific git refs, with specific base images.  Could be a future enhancement.

Reply:
Will move this to an enhancement ticket.